src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -365,34 +385,39 @@ protected void writeHashStoreYaml(String yamlString) throws IOException {
     protected String buildHashStoreYamlString(
         int storeDepth, int storeWidth, String storeAlgorithm, String storeMetadataNamespace
     ) {
```

Comment:
Once this PR is approved, I would like to take the opportunity to apply IntelliJ formatting to the codebase.

The reason for the inconsistency is because `hashstore-java` was first developed on VSCode, then IntelliJ - causing the formatting inconsitencies.